federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -75,6 +75,8 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 			rrset.rrsType = rrstype.CNAME
 		case ip.To4() != nil:
 			rrset.rrsType = rrstype.A
+		default:
```

Comment:
I was thinking it wasn't necessary, because it looks like ParseIP will either eval the string to an IP (v4 or v6) or return nil. As a result, I figured that there are cases for nil and IPv4, so the value would have to be Ipv6, if those two cases didn't match. Is that acceptable?